src/database/models/session.ts

```diff
@@ -47,7 +47,7 @@ Session.init({
 });
 
 Session.belongsTo(User, {
-    foreignKey: 'user_id',
+    foreignKey: 'userId',
```

Comment:
Is this supposed to be the foreignKey in the database or on the class?

Reply:
Sequelize accepts both, but I would rather use our field since we might change database field name but we won't change the key we use in the code, fixed for other comment